README.md

```diff
@@ -2,18 +2,257 @@
+
+If you are on Windows, you need [Git Bash](https://git-scm.com/downloads). On other platforms, like Linux or MacOSX; you would require having [git](https://git-scm.com/downloads) installed in your machine. On Windows, you might need to restart your machine after Vagrant installation.
+
+Once you have installed these, you can choose to boot up one of the following environments. In future, we plan on installing the basic requirements as part of `zeus` as well.
+
+### Clojure
+
+### Java
+
+In the Java box we have provided the following packages :
+
+* Java 8 - v1.8
+* Maven 3
+* Gradle 2
+* Tomcat 8
+
+Additionally, Postgres database has been setup.
+
+Tomcat server is set to auto-start on port 8080. Now in this box, the port 8080 is mapped with Host, i.e your machine's port 9100. Thus, run anything on the tomcat server and it would be accessible on your machine in the address [http://localhost:9100](http://localhost:9100)
+
+### Node
+
+In the Node box we have provided the following packages :
+
+* [MongoDB](https://www.mongodb.com/) 3.0 server and client
+* [NVM](https://github.com/creationix/nvm) or Node Version Manager
+* [Node](https://nodejs.org) version 4.4.3 (Can be easily updated via `nvm install`)
+* [Bower](http://bower.io/)
+* [Gulp](http://gulpjs.com/)
+* [Yeoman](http://yeoman.io/) CLI
+* [Express](http://expressjs.com/)
+* Essential Nodeschool Workshopper modules
+	* [git-it](https://github.com/jlord/git-it)
+	* [learnyounode](https://github.com/workshopper/learnyounode)
+	* [how-to-npm](https://github.com/npm/how-to-npm)
+	* [learnyoumongo](https://github.com/evanlucas/learnyoumongo)
+	* [expressworks](https://github.com/azat-co/expressworks)
+
+In our vagrant configuration, the port 3000 is mapped with guest port 9200. Thus, if you start an express app on port 3000, you can access it via the address [http://localhost:9200](http://localhost:9200)
+
+We have also included a sample [Todo app](https://github.com/tastejs/todomvc/tree/master/examples/socketstream/) with Socket Stream, built by the awesome guys at [TodoMVC](https://github.com/tastejs/todomvc). To run it, do the following :
+
+```bash
+cd /vagrant # navigate to shared vagrant folder
+cd socketstream
+npm install
+bower install
+npm start
+```
+
+Now open [http://localhost:9200](http://localhost:9200) in your browser and voila! The todo app is working!
+
+### PHP
+
+The PHP box comes with the following packages:
+
+  * Apache 2.4 webserver
+  * PHP 5.5 with these extensions:
+    * php5-intl 
+	  * php5-pgsql 
+    * php5-mcrypt 
+    * php5-sqlite 
+    * php5-apcu 
+    * php5-cli
+    * php5-gd
+  * PostgreSQL server 9.3 (MySQL or MariaDB support may come later)
+  * PostgreSQL client 9.3
+  * phpunit 3.7
+
+The typical workflow imagined for this PHP box is the following :
```

Comment:
~~The typical workflow imagined for this PHP box is the following :~~ The typical workflow that was imagined for this PHP box: